### DISCUSSION TITLE
Remove Clasess

### DISCUSSION THREAD

**Discussion by Ministerico** [burningwave/core#29] [General]
Hi,

How can I delete a class created with the loadOrBuildAndDefine method from a ClassFactory?

**Comment by Roberto-Gentili** [comment]
You mean to unload the class from the classloader?

**Reply by Ministerico** [reply]
Yes

**Comment by Roberto-Gentili** [comment]
In Java you cannot unload a class from a classloader once it has been loaded. You can use a temporary classLoader on which you can load the generated classed by calling the method `useClassLoader` on the `LoadOrBuildAndDefineConfig` object

**Reply by Ministerico** [reply]
Thanks!
